### PR TITLE
added export and filters to txnotification and claim code.

### PR DESCRIPTION
**Pull request by eoliveros**

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -9,6 +9,13 @@
 from app_core import app, db
 from utils import generate_key
 
+from flask_admin.babel import lazy_gettext
```
Comment (djpnewton) [inline models.py]: move to other flask_admin imports

**Comment by djpnewton** [comment]
there are no authorizer/proposer/viewer roles

File: models.py [hunk 2]
```diff
@@ -1,14 +1,21 @@
 import datetime
 
 from flask import redirect, url_for, request
+from flask_admin.babel import lazy_gettext
+from flask_admin.contrib.sqla import tools
```
Comment (djpnewton) [inline models.py]: not used

File: models.py [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
+from flask_admin.contrib.sqla.filters import BaseSQLAFilter
+from flask_admin.model import filters
 from flask_security import Security, SQLAlchemyUserDatastore, \
     UserMixin, RoleMixin, login_required, current_user
 from flask_admin.contrib import sqla
 from marshmallow import Schema, fields
+from sqlalchemy.sql import not_, or_
```
Comment (djpnewton) [inline models.py]: not used

File: models.py [hunk 4]
```diff
@@ -1,14 +1,21 @@
...
+
 
 from app_core import app, db
 from utils import generate_key
 
+
```
Comment (djpnewton) [inline models.py]: remove

File: models.py (hunk 1 above)
Comment (eoliveros) [reply models.py]: can you give me an example of how?

File: models.py (hunk 2 above)
Comment (eoliveros) [reply models.py]: removed tis line:
from flask_admin.contrib.sqla import tools

File: models.py (hunk 3 above)
Comment (eoliveros) [reply models.py]: removed

File: models.py (hunk 4 above)
Comment (eoliveros) [reply models.py]: remove that one extra line

File: models.py (hunk 1 above)
Comment (eoliveros) [reply models.py]: moved this with the other flask_admin something.

File: models.py (hunk 4 above)
Comment (djpnewton) [reply models.py]: yes

**Comment by djpnewton** [comment]
remove create/delete from new model views

File: models.py [hunk 5]
```diff
@@ -9,6 +12,7 @@
...
```
Comment (djpnewton) [inline models.py]: unnecessary change

Comment (eoliveros) [reply models.py]: removed extra line

**Comment by djpnewton** [comment]
can you give better commit notes (and squash into a single commit)

**Comment by eoliveros** [comment]
I left the:
self.can_edit = True

in there. not sure if thats needed or it can be removed.

**Comment by djpnewton** [comment]
no remove the can_edit,

the commits are still not squashed into one commit

**Comment by djpnewton** [comment]
see #5
